### PR TITLE
gxkb: init at 0.9.0

### PR DESCRIPTION
**Pull request by omgbebebe**
###### Motivation for this change
I want to use this lightweight layout switcher on daily basis

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/gxkb/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchFromGitHub, autoreconfHook, pkgconfig, gtk3, libwnck3, libxklavier
```
Comment (siraben) [inline pkgs/applications/misc/gxkb/default.nix:1]:
```suggestion
{ lib, stdenv, fetchFromGitHub, autoreconfHook, pkg-config, gtk3, libwnck3, libxklavier
```

File: pkgs/applications/misc/gxkb/default.nix [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  version = "0.9.0";
+
+  src = fetchFromGitHub {
+    owner = "zen-tools";
+    repo = "gxkb";
+    rev = "v${version}";
+    sha256 = "1fmppvpfz8rip71agsc464fdz423qw0xy8i3pcic14cy5gcwh069";
+  };
+
+  nativeBuildInputs = [ pkgconfig autoreconfHook ];
```
Comment (siraben) [inline pkgs/applications/misc/gxkb/default.nix:16]:
```suggestion
  nativeBuildInputs = [ pkg-config autoreconfHook ];
```

File: pkgs/applications/misc/gxkb/default.nix [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchFromGitHub, autoreconfHook, pkgconfig, gtk3, libwnck3, libxklavier
+, appindicatorSupport ? false, libayatana-appindicator-gtk3
+}:
+
+stdenv.mkDerivation rec {
+  name = "gxkb-${version}";
```
Comment (siraben) [inline pkgs/applications/misc/gxkb/default.nix:6]:
```suggestion
  pname = "gxkb";
```

**Review by siraben — CHANGES_REQUESTED** [review]

File: pkgs/applications/misc/gxkb/default.nix (hunk 1 above)
Comment (omgbebebe) [reply pkgs/applications/misc/gxkb/default.nix]: thanks! fixed

File: pkgs/applications/misc/gxkb/default.nix (hunk 3 above)
Comment (omgbebebe) [reply pkgs/applications/misc/gxkb/default.nix]: thanks!

**Comment by siraben** [comment]
Please squash the commits.

**Comment by omgbebebe** [comment]
done.

**Review by siraben — APPROVED** [review]
Cannot build right now since I am on macOS, otherwise LGTM.

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 115323` at 1e08c477 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li><a href="https://gist.github.com/ead1009892156d7b8cbff00b7550be82">gxkb</a></li></li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md">unclear-gpl</a>

`gpl2` is a deprecated license, check if project uses `gpl2Plus` or `gpl2Only` and change `meta.license` accordingly.

Near pkgs/applications/misc/gxkb/default.nix:25:5:
```
   |
25 |     license = licenses.gpl2;
   |     ^
```
</li>
  </ul>
</details>

File: pkgs/applications/misc/gxkb/default.nix [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+  src = fetchFromGitHub {
+    owner = "zen-tools";
+    repo = "gxkb";
+    rev = "v${version}";
+    sha256 = "1fmppvpfz8rip71agsc464fdz423qw0xy8i3pcic14cy5gcwh069";
+  };
+
+  nativeBuildInputs = [ pkg-config autoreconfHook ];
+  buildInputs = [ gtk3 libwnck3 libxklavier ] ++ (lib.optional appindicatorSupport libayatana-appindicator-gtk3);
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/gxkb/default.nix:17]:
```suggestion
  buildInputs = [ gtk3 libwnck3 libxklavier ] ++ lib.optional appindicatorSupport libayatana-appindicator-gtk3;
```

File: pkgs/applications/misc/gxkb/default.nix [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+  configureFlags = [ (lib.optionalString appindicatorSupport  "--enable-appindicator=yes") ];
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/gxkb/default.nix:19]:
```suggestion
  configureFlags = lib.optional appindicatorSupport "--enable-appindicator=yes";
```

File: pkgs/applications/misc/gxkb/default.nix (hunk 4 above)
Comment (omgbebebe) [reply pkgs/applications/misc/gxkb/default.nix]: fixed

File: pkgs/applications/misc/gxkb/default.nix [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+    owner = "zen-tools";
+    repo = "gxkb";
+    rev = "v${version}";
+    sha256 = "1fmppvpfz8rip71agsc464fdz423qw0xy8i3pcic14cy5gcwh069";
+  };
+
+  nativeBuildInputs = [ pkg-config autoreconfHook ];
+  buildInputs = [ gtk3 libwnck3 libxklavier ] ++ lib.optional appindicatorSupport libayatana-appindicator-gtk3;
+
+  configureFlags = lib.optional appindicatorSupport  "--enable-appindicator=yes";
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/gxkb/default.nix:19]:
```suggestion
  configureFlags = lib.optional appindicatorSupport "--enable-appindicator=yes";
```

**Pull request merged**